### PR TITLE
Add description in English.

### PR DESCRIPTION
**Pull request by ykubota**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,32 +18,90 @@ NOT yet implemented all existing functions of HeapStats Analyzer.
...
+  * contrib: Additional Libraries
+    * RefTreePlugin: Reference Tree Plugin
+* heapstats-analyzer: Main program of HeapStatsFXAnalyzer
+* heapstats-cli: Commandline interface of HeapStats
 
 ## JDP ##
-* JDP のオートディスカバリ機能は、以下の起動引数が付与されて起動している JVM に対して行われます。
+
+* An automatic discovery function of JDP (Java Discovery Protocol) is enabled
+by JVM which is runned with the following options.
```
Comment (YaSuenag) [inline README.md]: Is "runned" correct? I think "run".

Comment (ykubota) [inline README.md]: oops. fixed it.

**Comment by YaSuenag** [comment]
+1

**Comment by ykubota** [comment]
:ok_hand:

**Pull request merged**